### PR TITLE
user feedback

### PR DESCRIPTION
**Pull request by bgruening**
ping @mblue9 @bebatut

### CODE REVIEW

File: _posts/website/_posts/2018-10-31-tiaas-feedback.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+site: freiburg
+title: 'User feedback from Maria Doyle'
```
Comment (erasche) [inline _posts/website/_posts/2018-10-31-tiaas-feedback.md]:
```suggestion
title: 'Training Infrastructure Feedback from Maria Doyle'
```

File: _posts/website/_posts/2018-10-31-tiaas-feedback.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+external: "https://www.petermac.org/"
```
Comment (erasche) [inline _posts/website/_posts/2018-10-31-tiaas-feedback.md]: I'm not sure we should use external here, that makes this  not a post but a redirect to their site. I don't think that makes sense in this case.

**Comment by erasche** [comment]
ping @erasche too? Nice to hear positive things about our infra sometimes :)

**Comment by bgruening** [comment]
> ping @erasche too? 

My assumption is you actually read the post from our website :)

**Comment by mblue9** [comment]
Cool thanks!

The link to Peter Mac doesn't work for me (clicking on it just gives me a blank page) but the link to the TIaaS page works.

**Comment by bgruening** [comment]
Fixed. Thanks Maria.

**Pull request merged**